website/pages/docs/k8s/service-sync.mdx

```diff
@@ -151,10 +154,18 @@ If an external IP list is present, a service instance in Consul will be created
 for each external IP. It is assumed that if an external IP is present that it
 is routable and configured by some other system.
 
+The service instance's port will be set to the *first* defined port of the
+service unless set specifically via the `consul.hashicorp.com/service-port` annotation (see [Service Ports](/docs/k8s/service-sync#service-ports)).
+
 #### ClusterIP
 
-ClusterIP services are synced by default as of `consul-k8s` version 0.3.0. In
-many Kubernetes clusters, ClusterIPs may not be accessible outside of the cluster,
+ClusterIP services are synced by default as of `consul-k8s` version 0.3.0.
+Each pod that is an endpoint for the service will be synced as a Consul service
```

Comment:
```suggestion
Each pod that has an endpoint for the service will be synced as a Consul service
```

Reply:
I think I did mean each pod that *is* an endpoint here of the service here. Why do you think it should be has?

Reply:
Oh, I see. Re-reading it again makes sense. Ignore this!